migrations/20231218165411_create_analytics_data_ga4.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function(knex) {
```

Comment:
we create a new table in our database to hold our GA4 data

Reply:
I see they are altering the UA table here, https://github.com/18F/analytics-reporter/blob/develop/migrations/20170522094056_rename_date_time_to_date.js, so we can just use `table.date("date")` ?